spdx-utils/src/main/kotlin/SpdxConstants.kt

```diff
@@ -60,6 +60,11 @@ object SpdxConstants {
      */
     const val LICENSE_LIST_URL = "https://spdx.org/licenses/"
 
+    /**
+     * The prefix to be used for SPDX document reference in the external document reference IDs.
+     */
+    const val DOC_REF_PREFIX = "DocumentRef-"
```

Comment:
Let's spell this out as `DOCUMENT_REF_PREFIX`.

Reply:
FYI, I decided to split this out to https://github.com/oss-review-toolkit/ort/pull/3576 as it's not strictly related to this PR and can already be used elsewhere in the existing code base.

Reply:
I'll rebase once your PR is merged

Reply:
I rebased the PR.